hierarchicalforecast/methods.py

```diff
@@ -1186,7 +1181,7 @@ def get_P_action(y):
                 (b.size, b.size), matvec=lambda v: R @ (S @ v)
             )
 
-            x_tilde, exit_code = sparse.linalg.bicgstab(A, b, atol="legacy")
+            x_tilde, exit_code = sparse.linalg.bicgstab(A, b)
```

Comment:
Do you remove this as SciPy no longer supports it? (I don't see it being mentioned as a valid value [on bicgstab page](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.linalg.bicgstab.html))

Reply:
There was an issue on Github with this too, I think this argument is indeed deprecated and needs to removed.